lib/netcore/Makefile.am

```diff
@@ -19,86 +19,25 @@
 
 SUBDIRS = . 
 
-THRIFT = $(top_builddir)/compiler/cpp/thrift
-
-TESTDIR = Tests/Thrift.PublicInterfaces.Compile.Tests
-GENDIR = $(TESTDIR)/gen-netcore
-
 THRIFTCODE = \
-			Thrift/Thrift.csproj \
```

Comment:
I don't think this is correct, removing all the dependencies here.  This means if a source code file changes but the project stays the same, make won't trigger a netcore build - it will think the target is up to date.  In the merge I am including both, and adding three missing source code files.  Because of this, I am going to have to submit a CI for the merge to make sure it works.

Reply:
.Net Core cli should automatically handle that - if file didn't change - it will not trigger the build - hope this will help you to spend less time on adding every file to list (autotools as I understand doesn't support wildcards)